guidelines/sc/22/focus-appearance-minimum.html

```diff
@@ -5,11 +5,11 @@ <h4>Focus Appearance (Minimum)</h4>
     <p class="conformance-level">AA</p>
 	<p class="change">New</p>
     
-    <p>For the keyboard focus indicator of each User Interface Component, all of the following are true:</p>
+    <p>When user interface components receive keyboard focus, all of the following are true:</p>
```

Comment:
Will need to be User Interface Component

Reply:
Someone tackled this in #1469, most are lower case so that PR was to make all lowercase.